src/actions/index.js

```diff
@@ -9,10 +9,16 @@ export const DECREMENT = 'DECREMENT';
 // logic of updating the central Redux store. That
 // is left to the reducer(s).
 
-export const increment = () => {
-  // Fill in this function
+export const increment = count => {
+  return {
+    type: INCREMENT,
+    payload: count + 1
+  }
```

Comment:
Another way to do this is by creating an action without a payload.
```
export const increment = () => {
  type: INCREMENT
}
```
You have the `count` value on your store state already, so sending count in a payload would be redundant though it still works.